src/pages/AccountMigration.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from "react";
```

Comment:
Just leave TODO comments that this stuff will be excised later, to make it easy on us to prune it all later